src/index.js

```diff
@@ -3,6 +3,9 @@ import PropTypes from 'prop-types'
 import ReactDOM, {unstable_renderSubtreeIntoContainer as renderSubtreeIntoContainer} from 'react-dom'
 import assign from 'object-assign'
 
+let FG_SIZE = 8
+let BG_SIZE = 9
```

Comment:
If these are not modified they should be `const`.

Reply:
Oups yes you right, I fixed that 👍